source/SupportLib/PlayGamesPluginSupport/src/main/java/com/google/games/bridge/ConnectionLifecycleCallbackProxy.java

```diff
@@ -12,21 +11,21 @@ public ConnectionLifecycleCallbackProxy(Callback callback) {
         this.callback = callback;
     }
 
-    public void onConnectionResult(@NonNull String endpointId, @NonNull ConnectionResolution resolution) {
```

Comment:
Probably it would be more clear to just replace them by commented ones.

Reply:
Replaced.